src/stories/containers/cu-table/cu-table.renders.tsx

```diff
@@ -121,3 +101,37 @@ export const renderLastModified = (coreUnit: CoreUnitDto) => {
   if (!coreUnit) return <CuTableColumnLastModified date={undefined} isLoading={!coreUnit} />;
   return <CuTableColumnLastModified date={getLastMonthWithData(coreUnit)} code={getShortCode(coreUnit.shortCode)} />;
 };
+
+const LinksContainer = styled.div({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  alignItems: 'flex-end',
+  paddingRight: '16px',
+  width: '100%',
+  height: '50px',
+  margin: 'auto 0',
+  cursor: 'pointer',
+  [lightTheme.breakpoints.up('desktop_1194')]: {
+    paddingRight: '0px',
+    height: 'fit-content',
+  },
+  '@media (min-width: 1440px)': {
```

Comment:
You should use `[lightTheme.breakpoints.up('desktop_1440')]` here for consistency in the code

Reply:
you are right, done